proto/agglayer/interop/types/v1/aggchain.proto

```diff
@@ -8,10 +8,40 @@ import "agglayer/interop/types/v1/bytes.proto";
 message AggchainData {
   // The data of the aggchain.
   oneof data {
-    // Signature committed to the bridge exits and imported bridge exits.
+    // Legacy ECDSA signature on a commitment to the state transition.
     FixedBytes65 signature = 1;
-    // Generic Aggchain proof.
+
+    // Aggchain proof with the single ECDSA verified by the agglayer only.
     AggchainProof generic = 2;
+
```

Comment:
Nit: could this field be renamed `generic` -> `proof_only` or something like that? I think this is a backwards compatible change at the wire encoding level since only the numbers are important there. May break things at source level though since the generated structs will have the updated field name.